src/v2/Apps/Order/Components/PaymentPicker.tsx

```diff
@@ -35,9 +41,14 @@ import {
   SystemContextConsumer,
   SystemContextProps,
 } from "v2/Artsy/SystemContext"
+import { createStripeWrapper } from "v2/Apps/Auction/Components/Form"
```

Comment:
(thought) Could we relocate this function to `v2/Utils` since its shared between a few a few different apps?

Reply:
Totally, I totally don't remember creating this file here anyway.